articles/machines/qc-trace-simulator/invalidated-qubits-use-checker.md

```diff
@@ -20,7 +20,7 @@ operation UseReleasedQubitTest () : Unit {
     body (...) {
```

Comment:
The change is correct. going further than that, if there is only one body (and no other specializations), one can specify the implementation directly within the operation declaration. Additionally, single qubits can be allocated by now (both of these things are optional, but it would be good for docs to illustrate how code should be written). with that the example then becomes

operation UseReleasedQubitTest () : Unit {
    mutable q = new Qubit[1];
    using(ans = Qubit()) {
        set q w/= 0 <- ans;
    }
    H(q[0]);
}